src/utils.js

```diff
@@ -21,6 +21,12 @@ export function isSameUser(currentMessage = {}, diffMessage = {}) {
 
 }
 
+export function isFromOtherUser({ currentMessage = {}, user = {} }) {
+
+  return currentMessage.fromSystem || (currentMessage.user._id !== user._id);
```

Comment:
I'm not sure that is a good idea to use `currentMessage.fromSystem` here. Semantically, the function is checking is the given message is from a user different to given one. Even when probably a message from the system will be treated as a message _from other user_ (in fact, it doesn't have user), I believe that is out of the scope of this function to check it.

I prefer to move it off the function and keep it in the original `if` statement, but once again advices from @kfiroo  are welcome. In the end, he is who decides if this feature will be included (or not) and in what way.